model/File.ts

```diff
@@ -0,0 +1,24 @@
+import * as fsPromise from "fs";
+import { User } from "./User";
+
+export class File {
+  private static instance: File;
+
+  private constructor() {}
```

Comment:
Is this required?

Reply:
To prevent object formation constructor must be private.